simd/src/Kokkos_SIMD_NEON.hpp

```diff
@@ -453,6 +453,183 @@ simd<double, simd_abi::neon_fixed_size<2>> min(
+  }
+  [[nodiscard]] KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION friend mask_type
+  operator<(simd const& lhs, simd const& rhs) noexcept {
+    return mask_type(vclt_f32(lhs.m_value, rhs.m_value));
+  }
+  [[nodiscard]] KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION friend mask_type
+  operator<=(simd const& lhs, simd const& rhs) noexcept {
+    return mask_type(vcle_f32(lhs.m_value, rhs.m_value));
+  }
+  [[nodiscard]] KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION friend mask_type
+  operator>=(simd const& lhs, simd const& rhs) noexcept {
+    return mask_type(vcge_f32(lhs.m_value, rhs.m_value));
+  }
+  [[nodiscard]] KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION friend mask_type
+  operator!=(simd const& lhs, simd const& rhs) noexcept {
+    return !(lhs == rhs);
+  }
+};
+
+KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
+simd<float, simd_abi::neon_fixed_size<2>> abs(
+    simd<float, simd_abi::neon_fixed_size<2>> const& a) {
+  return simd<float, simd_abi::neon_fixed_size<2>>(
+      vabs_f32(static_cast<float32x2_t>(a)));
+}
+
+KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
+simd<float, simd_abi::neon_fixed_size<2>> copysign(
+    simd<float, simd_abi::neon_fixed_size<2>> const& a,
+    simd<float, simd_abi::neon_fixed_size<2>> const& b) {
+  uint32x2_t const sign_mask = vreinterpret_u32_f32(vmov_n_f32(-0.0));
+  return simd<float, simd_abi::neon_fixed_size<2>>(vreinterpret_f32_u32(
+      vorr_u32(vreinterpret_u32_f32(static_cast<float32x2_t>(abs(a))),
+               vand_u32(sign_mask,
+                        vreinterpret_u32_f32(static_cast<float32x2_t>(b))))));
+}
+
+KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
+simd<float, simd_abi::neon_fixed_size<2>> sqrt(
+    simd<float, simd_abi::neon_fixed_size<2>> const& a) {
+  return simd<float, simd_abi::neon_fixed_size<2>>(
+      vsqrt_f32(static_cast<float32x2_t>(a)));
+}
+
+KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
+simd<float, simd_abi::neon_fixed_size<2>> fma(
+    simd<float, simd_abi::neon_fixed_size<2>> const& a,
+    simd<float, simd_abi::neon_fixed_size<2>> const& b,
+    simd<float, simd_abi::neon_fixed_size<2>> const& c) {
+  return simd<float, simd_abi::neon_fixed_size<2>>(
+      vfma_f32(static_cast<float32x2_t>(c), static_cast<float32x2_t>(b),
+               static_cast<float32x2_t>(a)));
+}
+
+KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
+simd<float, simd_abi::neon_fixed_size<2>> max(
+    simd<float, simd_abi::neon_fixed_size<2>> const& a,
+    simd<float, simd_abi::neon_fixed_size<2>> const& b) {
+  return simd<float, simd_abi::neon_fixed_size<2>>(
+      vmax_f32(static_cast<float32x2_t>(a), static_cast<float32x2_t>(b)));
+}
+
+KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
+simd<float, simd_abi::neon_fixed_size<2>> min(
+    simd<float, simd_abi::neon_fixed_size<2>> const& a,
+    simd<float, simd_abi::neon_fixed_size<2>> const& b) {
+  return simd<float, simd_abi::neon_fixed_size<2>>(
+      vmin_f32(static_cast<float32x2_t>(a), static_cast<float32x2_t>(b)));
+}
+
+[[nodiscard]] KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
```

Comment:
Same comment as in AVX2 about the `[[nodiscard]]` attribute here, the ordering of the overloaded operators above, and the unnecessary defaulting of the special members.

Reply:
Fixed in 30aa0e7.